pennylane/ops/mid_measure/pauli_measure.py

```diff
@@ -117,7 +117,54 @@ def _pauli_measure_primitive_abstract_eval(*_, **__):
 
 
 def pauli_measure(pauli_word: str, wires: WiresLike, postselect: int | None = None):
-    """Perform a Pauli product measurement."""
+    """Perform a Pauli product measurement.
+
```

Comment:
Can you add a brief mathematical explanation of what a PPM does? Essentially a Pauli product measurement is the measurement of a tensor product of Pauli observables. The eigenvalue of a Pauli product is 1 or -1, and they map to 0 and 1 in the outcome of the PauliMeasure, and the state collapses to the superpositions of all degenerate eigenstates corresponding to the measured eigenvalue.